packages/trpc/server/routers/viewer/organizations/listMembers.handler.ts

```diff
@@ -230,7 +236,7 @@ export const listMembersHandler = async ({ ctx, input }: GetOptions) => {
 
       return {
         id: user.id,
-        username: user.username,
+        username: user.profiles[0].username || user.username,
```

Comment:
This could break if there's no profiles
```suggestion
        username: user.profiles[0]?.username || user.username,
```